w3af/plugins/crawl/wpscan/update.py

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/env python
+
+"""
+update.py
+
+Copyright 2015 Andres Riancho
+
+This file is part of w3af, http://w3af.org/ .
+
+w3af is free software; you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation version 2 of the License.
+
+w3af is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with w3af; if not, write to the Free Software
+Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA  02110-1301  USA
+
+"""
+import os
+import re
+import sys
+import subprocess
+
+
+URL = 'http://plugins.svn.wordpress.org/'
+PLUGIN_FILE = 'w3af/plugins/crawl/wpscan/plugins.txt'
```

Comment:
Please write a unittest that makes sure that the plugins file has been updated in the last month.